tests/Issues/GetValueMagicDir/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\Issues\GetValueMagicDir\Fixture;
+
+class Fixture
+{
+    public function run()
+    {
+        strlen(__DIR__);
+        strlen(__FILE__);
+
+        new \Foo(['bar' => ['foo' => __DIR__ . '/bar']]);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Issues\GetValueMagicDir\Fixture;
+
+class Fixture
+{
+    public function run()
+    {
+        'tests/Issues/GetValueMagicDir/Fixture';
```

Comment:
why is `strlen(__DIR__);` turned into `'tests/Issues/GetValueMagicDir/Fixture';`?

should it be `strlen('tests/Issues/GetValueMagicDir/Fixture');` instead?